forwards_paper.tex

```diff
@@ -449,32 +439,33 @@ \subsection*{The \msprime\ Tables API}
 of the \msprime\ Python API, which provides a powerful platform for processing
 tree topology and mutation data. The new portions of \msprime{} that we discuss
 here are dedicated to tree sequence input and output using simple tables of
-data, so we refer to this as the `Tables API'. The four key tables (nodes,
-edges, sites and mutations) are described above.
+data, as described above, so we refer to this as the ``Tables API''.
 
 The Tables API is primarily designed to facilitate efficient interchange of
 data between programs or between different modules of the same program. We
-adopted a `columnar' design, i.e., where all the values for a
+adopted a `columnar' design, where all the values for a
 particular column are stored in adjacent memory locations.
-There are many advantages to columnar storage---for example, since adjacent
+There are many advantages to columnar storage -- for example, since adjacent
 values in memory are from the same column, they tend to compress well,
 and suitable encodings can be chosen on a per-column basis~\citep{abadi2006integrating}.
 A particular advantage of this approach is that it enables very
 efficient copying of data, and in principle zero-copy data access
-(where a data consumer reads directly from the memory of a producer) is possible.
-Our implementation uses the NumPy C API~\citep{walt2011numpy} to efficiently copy
-data from Python into the low-level C library used to manipulate tree sequences.
+(where a data consumer reads directly from the memory of a producer).
+Our implementation 
+% uses the NumPy C API~\citep{walt2011numpy} to efficiently copy
```

Comment:
modified re Kevin's comment

Reply:
Msprime does use the numpy C api though. I'll update this to be clearer.

Reply:
Actually what's there is good, ignore this comment.